package.json

```diff
@@ -1,12 +1,11 @@
 {
   "name": "handsome-trello",
-  "version": "0.8.1",
+  "version": "0.8.2",
```

Comment:
While I don't think it's worth holding things up for, this isn't actually a part of the "parent child layout misplaced" issue, and therefore shouldn't be part of this branch. (It seems a little pedantic, I know, but it pays off when you have to roll back a feature or choose not to include it in a given release.) Normally, per gitflow, we'd make a "release/0.8.2" branch off `develop` when we're ready to do a release, make these sorts of changes there, and eventually merge that branch back into both `develop` and `master`.

Reply:
Good point. Should I revert this change?